src/main/java/org/jitsi/jicofo/util/JingleOfferFactory.java

```diff
@@ -539,6 +582,13 @@ private static void addAudioToContent(ContentPacketExtension content,
         // fmtp:111 useinbandfec=1
         addParameterExtension(opus, "useinbandfec", "1");
 
+        if (ENABLE_TCC)
```

Comment:
could we just pass this in (like `stereo`) and keep this method static?

Reply:
ENABLE_TCC looks like a static final but is a final instance field. I can rename the field to lower-case (enableTcc), keep the method static with an extra param, and overload it with an instance method that will pass the enableTcc field. Would that work?

Reply:
sounds good!